infra/nativetest/src/test/java/org/apache/shardingsphere/infra/nativetest/jdbc/features/repository/AddressRepository.java

```diff
@@ -0,0 +1,133 @@
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.infra.nativetest.jdbc.features.repository;
+
+import org.apache.shardingsphere.infra.nativetest.jdbc.features.entity.Address;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.LinkedList;
+import java.util.List;
+
+@SuppressWarnings({"SqlDialectInspection", "SqlNoDataSourceInspection"})
+public final class AddressRepository {
+    
+    private final DataSource dataSource;
+    
+    public AddressRepository(final DataSource dataSource) {
+        this.dataSource = dataSource;
+    }
+    
+    /**
+     * create table t_address if not exists.
+     * @throws SQLException SQL exception
+     */
+    public void createTableIfNotExists() throws SQLException {
+        String sql = "CREATE TABLE IF NOT EXISTS t_address (address_id BIGINT NOT NULL, address_name VARCHAR(100) NOT NULL, PRIMARY KEY (address_id))";
+        try (
+                Connection connection = dataSource.getConnection();
+                Statement statement = connection.createStatement()) {
+            statement.executeUpdate(sql);
+        }
+    }
+    
+    /**
+     * drop table t_address.
+     * @throws SQLException SQL exception
+     */
+    public void dropTable() throws SQLException {
+        String sql = "DROP TABLE IF EXISTS t_address";
+        try (
+                Connection connection = dataSource.getConnection();
+                Statement statement = connection.createStatement()) {
+            statement.executeUpdate(sql);
+        }
+    }
+    
+    /**
+     * truncate table t_address.
+     * @throws SQLException SQL exception
+     */
+    public void truncateTable() throws SQLException {
+        String sql = "TRUNCATE TABLE t_address";
+        try (
+                Connection connection = dataSource.getConnection();
+                Statement statement = connection.createStatement()) {
+            statement.executeUpdate(sql);
+        }
+    }
+    
+    /**
+     * insert something to table t_address.
+     * @param address address
+     * @return addressId of the insert statement
+     * @throws SQLException SQL exception
+     */
+    public Long insert(final Address address) throws SQLException {
+        String sql = "INSERT INTO t_address (address_id, address_name) VALUES (?, ?)";
+        try (
+                Connection connection = dataSource.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(sql)) {
+            preparedStatement.setLong(1, address.getAddressId());
+            preparedStatement.setString(2, address.getAddressName());
+            preparedStatement.executeUpdate();
+        }
+        return address.getAddressId();
+    }
+    
+    /**
+     * delete by id.
+     * @param id id
```

Comment:
Are these comments necessary?

Reply:
- This is a requirement from the checkstyle plugin.  I can't seem to find a simple action to block the check.  This submodule is only used for writing unit tests.

Reply:
OK, I see, thank you!